pkgs/by-name/lo/looking-glass-client/package.nix

```diff
@@ -132,7 +114,6 @@ stdenv.mkDerivation (finalAttrs: {
 
   postInstall = ''
     mkdir -p $out/share/pixmaps
-    ln -s ${desktopItem}/share/applications $out/share/
```

Comment:
Is there a reason why you removed the desktop entry?

Reply:
They have their own now https://github.com/gnif/LookingGlass/blob/B7/resources/looking-glass-client.desktop.in